scripts/get_das_info.py

```diff
@@ -76,13 +174,13 @@ def print_das_info(das_strings: list[str], keys_of_interest: tuple | None = None
                     info_of_interest["nfiles"] = dataset_info.get("nfiles", "")
                     info_of_interest["nevents"] = dataset_info.get("nevents", "")
 
-            desired_output = convert_to_desired_structure(info_of_interest)
+            desired_output = convert_function(info_of_interest)
             print(desired_output)
-            print()
 
 
 if __name__ == "__main__":
     parser = ArgumentParser()
     parser.add_argument("-d", "--dataset", dest="dataset", nargs="+", help="das name")
+    parser.add_argument("-f", "--function", dest="function", help="function that converts info into code")
```

Comment:
Having the possibility to use a custom conversion function looks very useful, but I'm thinking that maybe the option name `--function` is too generic. Not sure I have a better suggestion here though. Maybe `--conversion` or simply `--convert`?

In any case, it would be useful to add a default conversion function here (e.g. `default="convert_to_desired_structure"`) and also add `choices=list(convert_functions)` to get more info on the command line about what conversion functions are available.

Reply:
done, changed it to `--convert`